src/experimental/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,296 @@
+import React, {Children, useEffect, useRef, useState} from 'react'
+import Box from '../../Box'
+import sx, {SxProp} from '../../sx'
+import {useId} from '../../hooks/useId'
+import {invariant} from '../../utils/invariant'
+import {warning} from '../../utils/warning'
+import styled from 'styled-components'
+import {get} from '../../constants'
+import {ComponentProps} from '../../utils/types'
+import {getAnchoredPosition} from '@primer/behaviors'
+import type {AnchorSide, AnchorAlignment} from '@primer/behaviors'
+import '@oddbird/popover-polyfill'
+
+const StyledTooltip = styled.div`
+  // tooltip element itself
+  position: relative;
+  padding: 0.5em 0.75em;
```

Comment:
I'm going to leave suggestions with CSS variables since we plan to move to CSS 😄
 
```suggestion
  padding: var(--base-size-8) var(--base-size-4);
```

Hope I don't have those backwards